server/community.go

```diff
@@ -64,3 +66,62 @@ func (s *Server) postPRWelcomeMessage(ctx context.Context, pr *model.PullRequest
 
 	return nil
 }
+
+func (s *Server) assignGreeter(ctx context.Context, pr *model.PullRequest, repo *Repository) error {
+	// Only assign an Usher for community PRs
```

Comment:
```suggestion
	// Only assign an greeter for community PRs
```

Reply:
haha, that was the original name, but I didn't like it for looking too similar to user, which might end up in people misreading later on. thanks for catching that one.